### DISCUSSION TITLE
Very long table 4 compute time (4 times longer as normal)

### DISCUSSION THREAD

**Discussion by tkmbe99** [Chia-Network/chia-blockchain#6285] [General]
Hello

I almost canceled my plot because creating table 4 tooks me 14681 seconds (4h 4min) instead of 3310.305 seconds (55min).
Why is this possible? My plots needs normaly only 1h.
The cpu was only plotting as always.
I gave 6000MB and 4 threads (I hav 16GB and 4 core), the same worked nice before.
Any idea why could this be?

**Comment by kettlewarrior** [comment]
Hi,

I'd go for resource contention, swap use, or a disk that is giving up.

Did / does restarting your machine fix the problem?

Which operating system?

**Comment by room101-dev** [comment]
> Hello
> 
> I almost canceled my plot because creating table 4 tooks me 14681 seconds (4h 4min) instead of 3310.305 seconds (55min).
> Why is this possible? My plots needs normaly only 1h.
> The cpu was only plotting as always.
> I gave 6000MB and 4 threads (I hav 16GB and 4 core), the same worked nice before.
> Any idea why could this be?

When U post a problem like this U must tell us what HW your running, there is MINIMAL hardware. Given the symptoms, and assuming your using the correct hw to plot, ...

**Your plotting on NVME right? You need to run "Sudo fstrim -av" daily to clean your NVME drive otherwise it will slow down**

[ nvme drives don't houseclean, they don't zero the data post delete, fstrim cleans the house, and makes the NVME fresh&clean, don't worry this is an OS tool, its not chia-dev scamware ]

Cmd line is "Chia plots create -t /dev/nvme -2 /dev/sss-sata -d /dev/hdd" that's it

If you don't have -t nvme, and -2 ssd ( both > 1TB ) then your going to go nowhere quick, remember it takes 800 plots ( 80TB ) earn 1 xch per month and at 800, with the 'space' EB doubling every 9 days ( 8%/day) your plotting reward halves every week, last week 400 plots earned me 1 xch month.

**One plot was 7 cents per  day last week, this week its 3.5 cents, next week its 1.75 cents.**
